opentracing-api/src/main/java/io/opentracing/ScopeManager.java

```diff
@@ -48,7 +39,7 @@
      *
      * <p>
      * If there is an {@link Scope non-null scope}, its wrapped {@link Span} becomes an implicit parent of any
-     * newly-created {@link Span} at {@link Tracer.SpanBuilder#startActive()} time (rather than at
+     * newly-created {@link Span} at {@link Tracer.SpanBuilder#startActive(boolean)} time (rather than at
      * {@link Tracer#buildSpan(String)} time).
```

Comment:
- not just `startActive` but `start` as well.
- no need for parentheses around `(rather than at buildSpan)`